app.js

```diff
@@ -1,474 +1,570 @@
+  };
+
+  ajaxRequest(
+    "POST",
+    comment,
+    `https://api.github.com/repos/${urlObj.organization}/${
+      urlObj.currentRepo
+    }/issues/${urlObj.issueNumber}/comments`
+  ).then(response => {
     if (closeOriginal) {
       // if success, close the existing issue and open new in a new tab
-      closeGithubIssue(oldIssue)
+      closeGithubIssue(oldIssue);
     }
-    goToIssueList(repo, newIssue.number, urlObj.organization, urlObj.currentRepo)
-  })
+    goToIssueList(
+      repo,
+      newIssue.number,
+      urlObj.organization,
+      urlObj.currentRepo
+    );
+  });
 }
 
 function goToIssueList(repo, issueNumber, org, oldRepo) {
   // based on user settings, determines if the issues list will open after a clone or not
-  chrome.runtime.sendMessage({ repo: repo, issueNumber: issueNumber, organization: org, oldRepo: oldRepo }, (response) => {
-  })
+  chrome.runtime.sendMessage(
+    {
+      repo: repo,
+      issueNumber: issueNumber,
+      organization: org,
+      oldRepo: oldRepo
+    },
+    response => {}
+  );
 }
 
 function ajaxRequest(type, data, url) {
   return new Promise((resolve, reject) => {
-    chrome.storage.sync.get({
-      githubToken: ''
-    }, (item) => {
-      token = item.githubToken
-      $.ajax({
-        type: type,
-        beforeSend: (request) => {
-          request.setRequestHeader('Authorization', 'token ' + token)
-          request.setRequestHeader('Content-Type', 'application/json')
-        },
-        data: JSON.stringify(data),
-        url: url
-      }).done((data, status, header) => {
-        resolve({
-          data: data,
-          status: status,
-          header: header
-        })
-      })
-    })
-  })
+    chrome.storage.sync.get(
+      {
+        githubToken: ""
+      },
+      item => {
+        token = item.githubToken;
+        $.ajax({
+          type: type,
+          beforeSend: request => {
+            request.setRequestHeader("Authorization", `token ${token}`);
+            request.setRequestHeader("Content-Type", "application/json");
+          },
+          data: JSON.stringify(data),
+          url: url
+        }).done((data, status, header) => {
+          resolve({
+            data: data,
+            status: status,
+            header: header
+          });
+        });
+      }
+    );
+  });
 }
 
 function addRepoToList(repoFullName, repo, section) {
   // add the repo to the list
-  const periodReplace = repo.replace(/\./g, '_')
+  const periodReplace = repo.replace(/\./g, "_");
 
   // determine where the item needs to go
-  if(section === 'used') {
-    if($('#' + periodReplace).length === 0) {
-      $('.dropdown-header-rest').before('<li data-toggle="modal" id="' + periodReplace + '" data-target="#kaminoModal"><a class="repoItem" href="#">' + repoFullName + '</a></li>')
+  if (section === "used") {
+    if ($(`# ${periodReplace}`).length === 0) {
+      $(`.dropdown-header-rest).before(
```

Comment:
the backticks aren't lining up here. Should be:

`.dropdown-header-rest`(which can actually just stay as '.dropdown-header-rest'. Then inside the `before`, `<li data-toggle... etc